lib/galaxy/webapps/galaxy/config_schema.yml

```diff
@@ -551,18 +569,20 @@ mapping:
 
       builds_file_path:
         type: str
-        default: tool-data/shared/ucsc/builds.txt
```

Comment:
Similarly, this could be `default: shared/ucsc/builds.txt`, to be interpreted w.r.t. `self.tool_data_path`

Same goes for `len_file_path`

Reply:
Same as above.

Reply:
These two done.